djangoapp/README.md

```diff
@@ -4,9 +4,7 @@ Requirements:
  * any recent Python 3
    * macOS: ```brew install python```
    * CentOS: ```yum -y install epel-release ; yum -y update ; yum -y install python34```
- * Django itself
-   ```pip3 install --upgrade pip ; pip3 install Django```
- * AWS SDK
-   ```pip3 install boto3```
+ * Django & AWS SDK
+   ```pip3 install --upgrade pip ; pip3 install -r requirements.txt```
```

Comment:
Small thing, but I'd document this over two lines, just because it's a bit easier to read. I always include a `$` symbol too, to indicate it's something to be executed in a shell:

```
$ pip3 install --upgrade pip
$ pip3 install -r requirements.txt
```